desc/configuration.py

```diff
@@ -312,19 +325,58 @@ def __init__(
         else:
             raise TypeError("Got unknown pressure profile {}".format(pressure))
 
+        # default profile
+        if iota is None and current is None:
+            warnings.warn(
+                colored(
+                    "Must specify either iota or current. "
+                    + "Using default profile of current=0.",
+                    "yellow",
+                )
+            )
+            self._current = PowerSeriesProfile(
+                modes=np.array([0]), params=np.array([0]), name="current"
+            )
+        elif iota is not None and current is not None:
+            raise ValueError("Cannot specify both iota and current profiles.")
+
+        # iota
         if isinstance(iota, Profile):
             self.iota = iota
         elif isinstance(iota, (np.ndarray, jnp.ndarray)):
             self._iota = PowerSeriesProfile(
                 modes=iota[:, 0], params=iota[:, 1], name="iota"
             )
-        elif iota is None:
-            self._iota = PowerSeriesProfile(
-                modes=np.array([0]), params=np.array([0]), name="iota"
-            )
-        else:
+        elif iota is not None:
             raise TypeError("Got unknown iota profile {}".format(iota))
 
+        # current
+        if isinstance(current, Profile):
+            self.current = current
+        elif isinstance(current, (np.ndarray, jnp.ndarray)):
+            self._current = PowerSeriesProfile(
+                modes=current[:, 0], params=current[:, 1], name="current"
+            )
+        elif current is not None:
+            raise TypeError("Got unknown current profile {}".format(current))
+
+        # warning about odd profiles
+        if isinstance(self.pressure, PowerSeriesProfile):
```

Comment:
There's the possibility that they may not be using explicitly even profiles, but just manually setting the odd coeffs to zero, so that might be a better check here

Reply:
I can add this, but the default profile is `sym="auto"` which sets sym to true if the odd coeffs are 0.

Reply:
should be easy to change, just look at whatever logic is used in `sym=auto` in `PowerSeriesProfile` and add [that ](https://github.com/PlasmaControl/DESC/blob/0d2f9b854026cef033e66c10bb5e1ba2e5699649/desc/profiles.py#L558)here?

Reply:
I think @unalmis point is correct, the default for the profile is to be symmetric if only symmetric modes are given, I think this warning is fine for now. Eventually we might reconsider. We might also want to define some sort of "even"-ness for spline profiles etc.

Reply:
yea for spline's if we could have a rho=0 knot which is constrained to have zero derivative then this would be the "even-ness" we are looking for I think